### PR TITLE
Add EFS (elastic file system)

### PR DESCRIPTION
**Pull request by mariogluhakovic**
- Add EFS (elastic file system) that is needed for measurement storage

### CODE REVIEW

**Review by schwichti — CHANGES_REQUESTED** [review]
* Update section *Billable Resources and Services* in README.md
* Update section *Architecture* in README.md
* Update section *Encryption* in README.md
* Is there a backup solution for the EFS? (if no, create a new issue for this for a separate PR).

File: storages.tf [hunk 1]
```diff
@@ -0,0 +1,75 @@
+locals {
+  storage_subnets = tomap({
+    zone1 = module.vpc.private_subnets[0]
```
[the remark this answers is not in the document]
Comment (mariogluhakovic) [reply storages.tf]: Changed.

**Comment by mariogluhakovic** [comment]
> * Update section _Billable Resources and Services_ in README.md
> * Update section _Architecture_ in README.md
> * Update section _Encryption_ in README.md
> * Is there a backup solution for the EFS? (if no, create a new issue for this for a separate PR).

Readme is updated.
Backup solution is not needed since this is temporary storage for data analyze. The same data can be found inside the Minio storage and recreate by running the experiment.

File: README.md [hunk 2]
```diff
@@ -25,23 +25,26 @@ These files are exchanged via an non-public S3 bucket that can be read and writt
...
-| ------- | ----------- | ---------- | 
+| Service | Description | Mandatory? |
+| ------- | ----------- | ---------- |
 | Amazon Elastic Kubernetes Service | A Kubernetes cluster is required to run SIMPHERA. | Yes |
 | Amazon Virtual Private Cloud | Virtual network for SIMPHERA. | Yes |
 | Elastic Load Balancing | SIMPHERA uses a network load balancer. | Yes |
 | Amazon EC2 Auto Scaling | SIMPHERA automatically scales compute nodes if the capacity is exhausted. | Yes |
 | Amazon Relational Database | Project and authorization data is stored in Amazon RDS for PostgreSQL instances. | Yes |
 | Amazon Simple Storage Service | Binary artifacts are stored in an S3 bucket. | Yes |
+| Amazon Elastic File System | Binary artifact are temporary stored in EFS | Yes |
```
Comment (schwichti) [inline README.md:39]: Binary artifacts are stored temporarily in EFS.

File: AWSReferenceArchitecture.png
Comment (schwichti) [inline AWSReferenceArchitecture.png:1]: Please use either the old or new AWS icon set consistently.

**Review by schwichti — CHANGES_REQUESTED** [review]

File: AWSReferenceArchitecture.png
Comment (mariogluhakovic) [reply AWSReferenceArchitecture.png:1]: Done.

**Review by schwichti — APPROVED** [review]

**Pull request merged**